evento/src/views/Explore/index.js

```diff
@@ -4,21 +4,37 @@ import EventCard from '../../components/EventCard';
 import './Explore.css';
 
 class Explore extends Component {
-	componentWillMount() {
-		this.setState({ events: [] });
+	constructor(props) {
+		super(props);
+		this.state = { events: [] };
+	}
 
+	componentDidMount() {
 		fetch('/events')
 		.then(response => response.json())
 		.then(events => {
 			this.setState({ events: events });
 		});
 	}
+	
+	isVisible(event, filterQuery) {
+		const includedProps = [event.title, event.description, event.category.name, 
+			event.creator.name, event.location];
+
+		const cleaned = includedProps
```

Comment:
Oh true. Well cleaned is still a bad name I think. What about:

`const searchString = includedProps.join(' ').toLowerCase();`

and then just simply seachString.includes(filterQuery.toLowerCase()). join(' ') converts all elements in array to string (which is good behavior imo, since then it includes objects that are Number etc as well) and if element is null or undefined, it just returns empty string